js/scenes/Map.js

```diff
@@ -31,7 +32,18 @@ export class Map extends Component {
           latitudeDelta: 0.0922,
           longitudeDelta: 0.0421,
         }}
+        renderMarkers={() => (
+          <MapMarker
+            coordinate={{
+              latitude: 51.504444,
+              longitude: -0.086667,
+            }}
+            source={{
+              uri:'https://media-cdn.tripadvisor.com/media/photo-s/07/a2/b2/e6/shoreditch-street-art.jpg',
+            }}/>
+        )}
```

Comment:
Can't these just be `children` of the `<MagnetMapView>` instead of having a dedicated render function?